src/messaging/tests/TestSecureChannelMgr.cpp

```diff
@@ -0,0 +1,272 @@
+/*
+ *
+ *    Copyright (c) 2020-2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      This file implements unit tests for the ExchangeManager implementation.
```

Comment:
I would drop this `@file` comment: generally I have not found any of these `@file` comments valuable when reading code and they seem to have a high probability of being copy & pasted - like this one I believe it should not be ExchangeManager but rather SecureChannelManager.

A comment saying that 'TestSecureChannelMgr.cpp implements unit tests for SecureChannelManager' seems redundant.